### PR TITLE
Claim multiple

### PR DESCRIPTION
**Pull request by Orland0x**

### CODE REVIEW

File: test/Claim.t.sol [hunk 1]
```diff
@@ -7,41 +7,78 @@ contract BoostClaimTest is BoostTest {
...
+        signatures[2] = _generateClaimSignature(claim3);
+        signatures[3] = _generateClaimSignature(claim4);
+        signatures[4] = _generateClaimSignature(claim5);
+        boost.claimMultiple(claims, signatures);
+        assertEq(token.balanceOf(address(boost)), depositAmount - 5);
+        assertEq(token.balanceOf(claimer), 1);
+        assertEq(token.balanceOf(claimer2), 1);
+        assertEq(token.balanceOf(claimer3), 1);
+        assertEq(token.balanceOf(claimer4), 1);
+        assertEq(token.balanceOf(claimer5), 1);
```
Comment (pscott) [inline test/Claim.t.sol:71]: I think a little bit more spacing / comments would be helpful here too :)

File: src/Boost.sol [hunk 2]
```diff
@@ -9,13 +9,18 @@ import "./IBoost.sol";
...
 
     uint256 public nextBoostId = 1;
     mapping(uint256 => BoostConfig) public boosts;
     mapping(address => mapping(uint256 => bool)) public claimed;
 
+    struct Claim2 {
+        address recipient;
+        uint256 amount;
+    }
+
```
Comment (pscott) [inline src/Boost.sol:23]: Dead code?

**Review by pscott — APPROVED** [review]
utACK, minor comments to add I think

File: src/Boost.sol (hunk 2 above)
Comment (Orland0x) [reply src/Boost.sol:23]: yeah just a mock i forgot to remove

**Pull request merged**